examples/cell_tracking.py

```diff
@@ -0,0 +1,74 @@
+"""
+Cell tracking example
+=====================
+
+This example demonstrates how the track layer is used
+for visualizing cell tracking data, by displaying 2D + time dataset
+of cells with colored properties for track-id.
+
+Thanks to Dr. Alessia Ruggieri and Philipp Klein, Centre for Integrative Infectious
+Disease Research (CIID), University Hospital Heidelberg, Germany for the data.
+You can find the data on: https://doi.org/10.5281/zenodo.15597019
+
+.. tags:: visualization-advanced, colored 2D tracks
+"""
+from pathlib import Path
+
+import numpy as np
+import pooch
+import tifffile
+from trackastra.model import Trackastra
+from trackastra.tracking import ctc_to_napari_tracks, graph_to_ctc
+
+import napari
+
+# Create temporary directory
+tmp_dir = Path(pooch.os_cache('napari-cell-tracking-example'))
+tmp_dir.mkdir(parents=True, exist_ok=True)
+
+# Extract silver truth data for trackastra
+st_url = "https://zenodo.org/records/15852284/files/masks_pred.npz?download=1"
+st_path = pooch.retrieve(
+    url=st_url,
+    fname="masks_pred.npz",
+    known_hash=None,
+    path=pooch.os_cache("napari cell tracking example")
+)
+
+# Extract raw tif files for trackastra
+tif_url = "https://zenodo.org/records/17643282/files/01(1).zip?download=1"
+tif_file = pooch.retrieve(
+    url=tif_url,
+    fname="01(1).zip",
+    known_hash=None,
+    path=pooch.os_cache("napari_cell_tracking_example"),
+    processor=pooch.Unzip(),
+)
+
+# Load the downloaded masks
+masks_npz = np.load(st_path)
+masks = masks_npz['masks']
+
+# Sort through images and stack them
+imgs = sorted(tif_file.rglob("*.tif"))
```

Comment:
`tif_file` is just a list of strings here, so the `rglob` doesn't work. We can just use vanilla string sorting though, because the naming convention works with just alphabetical sort.

```suggestion
imgs = sorted(tif_file)
```